docs/connectors/io-crd-config/sink-crd-config.md

```diff
@@ -70,13 +70,20 @@ To use a secret in an environment variable in a Pod, follow these steps.
 2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
 3. Modify your image and or command line so that the program looks for values in the specified environment variables.
 
+Pulsar clusters supports using TLS or other authentication plugin for authentication.
+
+| Field | Description |
+| --- | --- |The authentication secret contains the `clientAuthenticationPlugin` and `clientAuthenticationParametersPath` fields.|
+| `tlsSecret` | The TLS secret contains the `use_tls`, `tls_allow_insecure`, `hostname_verification_enabled`, `tls_trust_cert_path` fields.|
+
```

Comment:
It seems that there is sth wrong with the table format
![Uploading image.png…]()

Reply:
I suggest to split this part into two sub sections.

- TLS Secret
| Field | Description |
| --- | --- |
| `tlsAllowInsecureConnection` | Allow insecure tls connection. |
| `tlsHostnameVerificationEnable` | Enable hostname verification. |
| `tlsTrustCertsFilePath` | tls trust cert file path. |

- Auth Secret
| Field | Description |
| --- | --- |
| `clientAuthenticationPlugin` | Client authentication plugin. |
| `clientAuthenticationParameters` | Client authentication params. |